Tasks/DotNetCoreInstallerV0/externals/get-os-platform.ps1

```diff
@@ -0,0 +1,18 @@
+function Get-Machine-Architecture()
+{
+    # possible values: AMD64, IA64, x86
+    return $ENV:PROCESSOR_ARCHITECTURE
```

Comment:
I assume we are going with the agent arch. If the agent is 32 bit on  a 64 bit machine; we will install 32 bit sdk

Reply:
Yes. we are doing same as what dotnet install scripts do